src/styles/tools/_mixins.scss

```diff
@@ -4,27 +4,59 @@ Tools - Mixins
 
 
 /* === Flexbox === */
-@mixin flex($direction,$center, $wrap: nowrap ) {
+@mixin flex($direction, $alignment, $wrap: nowrap ) {
   display: flex;
   flex-direction: $direction;
   flex-wrap: $wrap;
-  @if $center {
+  @if $alignment == center {
     align-items: center;
     justify-content: center;
+  } @else if $alignment == space {
+    align-items: center;
+    justify-content: space-around;
+  } @else if $alignment == end {
+    align-items: center;
+    justify-content: flex-end;
   }
 }
 
+
+/* === Font Palettes === */
+@mixin title {
+  font-size: $font-size-header;
+  font-family: $font-family-primary;
+  text-transform: uppercase;
+  letter-spacing: .1rem;
+}
+
+@mixin subtitle {
+  font-size: $font-size-header;
+  font-weight: bolder;
+  text-transform: uppercase;
+  letter-spacing: .1rem;
+}
+
+@mixin scaling-font {
+  font-size: calc(20px + (32 + 10) * ((100vw - 375px) / (1000 - 375)));
```

Comment:
I'm a little confused as to how this is working. I haven't worked with calc before